README.md

```diff
@@ -49,9 +49,15 @@ As stated in [this issue](https://github.com/vercel/next.js/issues/30078#issueco
 $ export NODE_OPTIONS=--openssl-legacy-provider
 ```
 
-## Creating a static page
+## Creating a static page in Markdown
```

Comment:
These are very detailed! Almost anyone can create pages by following these steps :)

Since we have the other header to be 'Creating a static page in vue' which shows how to create a page from scratch using just vue, can we tweak this title to be 'Creating a static page using Markdown'? What do you think?